src/test/java/com/iexec/sms/secret/compute/TeeTaskComputeSecretTest.java

```diff
@@ -43,85 +42,6 @@ private TeeTaskComputeSecret getRequesterSecret() {
                 .build();
     }
 
-    @Test
-    void shouldNotSaveSecretFromDefaultBuilder() {
```

Comment:
Why don't we keep same tests than before?

Reply:
Those constraints should now be enforced in `TeeTaskComputeSecretHeader` constructor, so these tests are not relevant anymore.

Reply:
OK, so I've removed the explicit checks. Please see https://github.com/iExecBlockchainComputing/iexec-sms/pull/142/commits/ada5841a688eaecbdb1cae2be5d638f024a8055b.